src/components/Layout/index.jsx

```diff
@@ -17,24 +17,32 @@ class Layout extends React.PureComponent {
     meta: PropTypes.objectOf(PropTypes.string),
     marginTop: PropTypes.bool,
     footer: PropTypes.bool,
+    descriptionContent: PropTypes.string,
   }
 
   render() {
-    const { children, title, meta, marginTop, footer } = this.props
+    const {
+      children,
+      title,
+      marginTop,
+      footer,
+      descriptionContent,
+    } = this.props
 
     const classMain = classNames({
       [styles.main]: marginTop,
     })
 
     return (
       <React.Fragment>
-        <Helmet
-          title={`${title} | Codea`}
-          meta={[
-            { name: 'description', content: meta.description },
-            { name: 'keywords', content: meta.keywords },
-          ]}
-        />
+        <Helmet>
+          <title>{title ? `${title} | Codea` : `Codea`}</title>
+          <meta name="twitter:site" content="@wootsbot" />
+          <meta name="og:type" content="website" />
+          <meta name="og:site_name" content="Codea" />
+          <meta name="Description" content={`${descriptionContent}`} />
```

Comment:
No sé si sea camel case; no creo, pero en cualquier caso, tratándose de meta información, sugiero utilizar solo minúsculas:

```suggestion
          <meta name="description" content={`${descriptionContent}`} />
```

Reply:
Ya teniendo este dato, valuar agregarlo de una vez como `og:description` y `twitter:description`; digo valuarlo porque alguna vez lei que estos deberían ser tomados del `title` 🤷‍♂️